Week-4/Assi_3.js

```diff
@@ -0,0 +1,27 @@
+const Fib = {
```

Comment:
great work on this function, just give some meaningful name to function, like fibonaccifunction